src/plugins/intel_cpu/src/emitters/snippets/x64/jit_brgemm_emitter.hpp

```diff
@@ -6,24 +6,61 @@
 
 #include "emitters/plugin/x64/jit_emitter.hpp"
 #include "jit_kernel_emitter.hpp"
-
+#include "emitters/snippets/cpu_kernel_executor_table.hpp"
 #include <cpu/x64/brgemm/brgemm.hpp>
 
 namespace ov {
 namespace intel_cpu {
+class BrgemmKernelExecutor;
```

Comment:
On the contrary, SnippetExecutor is declared inside Snippet node [here](https://github.com/openvinotoolkit/openvino/blob/a8331cac613d99a2ead487b32c9ba01fc8cf8df2/src/plugins/intel_cpu/src/nodes/subgraph.h#L74-L84)

I don't think it matters much now, since we have a single kernel_executor (for brgemm). But I support creating a dedicated dir, when we'll have multiple kernel executors.

Reply:
I meant not exact `Snippet` node but other nodes that are often updated in plugin. There are many [executors](https://github.com/openvinotoolkit/openvino/tree/master/src/plugins/intel_cpu/src/nodes/executors).
Now it's really better! At least for me :blush: